tests/Fixtures/schemas/schema.xml

```diff
@@ -89,7 +89,6 @@
             <reference local="customer_id" foreign="id"/>
         </foreign-key>
         <foreign-key foreignTable="bookstore_contest" onDelete="cascade">
-            <reference local="bookstore_id" foreign="bookstore_id"/>
```

Comment:
@rizwanjiwan What was your reasoning to remove this? This apparently made the CI error go away, but I'm wondering what you figured out about it.

Reply:
@rizwanjiwan it looks like the foreign key you removed isn't used. I've addressed the issue in PR #2021 by adding the unique index this foreign key needs so it doesn't need to be removed.

Reply:
Sounds good